### PR TITLE
update zlib to v1.3.0

### PR DESCRIPTION
**Pull request by gangatp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,5 +44,6 @@ lib3mf is released under the [BSD license](LICENSE). The library contains code o
 1. libzip 1.7.3.1, https://libzip.org/license/
 2. zlib 1.2.11, http://www.zlib.net/zlib_license.html
```
Comment (martinweismann) [inline README.md:45]: version is incorrect here

File: Libraries/zlib/zlib_v1.3^0.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+"v1.3^0"
```
Comment (martinweismann) [inline Libraries/zlib/zlib_v1.3^0.txt:1]: where does this caret "^" come from?
I would have expected "v1.3.0".

**Review by martinweismann — APPROVED** [review]

File: Libraries/zlib/zlib_v1.3^0.txt (hunk 2 above)
Comment (gangatp) [reply Libraries/zlib/zlib_v1.3^0.txt:1]: That's how the git tag is defined. I write git tag as text file to know from which version of this third party library is copied from.

Comment (martinweismann) [reply Libraries/zlib/zlib_v1.3^0.txt:1]: I do not quite understand: This is the tag, right?
https://github.com/madler/zlib/releases/tag/v1.3 and `git tag` simply returns 
```
v0.71
v0.79
v0.8
v0.9
v0.91
v0.92
v0.93
v0.94
v0.95
...
v1.2.6.1
v1.2.7
v1.2.7.1
v1.2.7.2
v1.2.7.3
v1.2.8
v1.2.9
v1.3
```

Comment (gangatp) [reply Libraries/zlib/zlib_v1.3^0.txt:1]: ah I use **git describe --contains HEAD** to the text file, which returns v1.3^0.  When I was using git tag sometimes that was empty, so used git describe https://github.com/gangatp/lib3mf/blob/d1817bebc4a551d53a9494ff7284c87a6cb033bc/submodules/update_libs.sh#L22

Comment (martinweismann) [reply Libraries/zlib/zlib_v1.3^0.txt:1]: i see.

Should be fine, I guess

File: README.md (hunk 1 above)
Comment (gangatp) [reply README.md:45]: will change it

**Comment by gangatp** [comment]
duplicate of https://github.com/3MFConsortium/lib3mf/pull/336

**Pull request closed without merging**